src/Input.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import Input from './Input';
+
+test('Input', () => {
+  const value = {
+    id: 1,
+    text: '할 일 1',
+  };
+  const handleClick = jest.fn();
+  const handleChange = jest.fn();
+
+  const { container, getByText } = render(
+    <Input value={value} onClick={handleClick} onChange={handleChange} />
+  );
```

Comment:
```suggestion
  const { container, getByText } = render((
    <Input 
      value={value}
      onClick={handleClick}
      onChange={handleChange}
     />
  ));
```

처럼 줄바꿈을 해줘도 코드 가독성에 좋을 것 같아요 ;)